modules/PhyVersoBSP/connector_2/evse_board_supportImpl.cpp

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#include "evse_board_supportImpl.hpp"
+
+namespace module {
+namespace connector_2 {
+
+void evse_board_supportImpl::init() {
+    {
+        std::scoped_lock lock(caps_mutex);
+
+        caps.min_current_A_import = mod->config.conn2_min_current_A_import;
+        caps.max_current_A_import = mod->config.conn2_max_current_A_import;
+        caps.min_phase_count_import = mod->config.conn2_min_phase_count_import;
+        caps.max_phase_count_import = mod->config.conn2_max_phase_count_import;
+        caps.supports_changing_phases_during_charging = false;
+
+        caps.min_current_A_export = mod->config.conn2_min_current_A_export;
+        caps.max_current_A_export = mod->config.conn2_max_current_A_export;
+        caps.min_phase_count_export = mod->config.conn2_min_phase_count_export;
+        caps.max_phase_count_export = mod->config.conn2_max_phase_count_export;
+        caps.supports_changing_phases_during_charging = false;
+
+        if (mod->config.conn2_has_socket) {
+            caps.connector_type = types::evse_board_support::Connector_type::IEC62196Type2Socket;
+        } else {
+            caps.connector_type = types::evse_board_support::Connector_type::IEC62196Type2Cable;
+        }
+    }
+
+    mod->serial.signal_cp_state.connect([this](int connector, CpState s) {
+        if (connector == 2 and s not_eq last_cp_state) {
+            publish_event(to_bsp_event(s));
+            EVLOG_info << "[2] CP State changed: " << to_bsp_event(s);
+            last_cp_state = s;
+        }
+    });
+    mod->serial.signal_set_coil_state_response.connect([this](int connector, CoilState s) {
+        if (connector == 1) {
+            EVLOG_info << "[1] Relais: " << (s.coil_state ? "ON" : "OFF");
+            publish_event(to_bsp_event(s));
+        }
```

Comment:
did we forget the telemetry connection for connector 2 or was it dropped on purpose?